tests/integration-tests/tests/common/utils.py

```diff
@@ -231,3 +235,33 @@ def get_route_tables(subnet_id, region):
         Filters=[{"Name": "association.subnet-id", "Values": [subnet_id]}]
     )
     return [table["RouteTableId"] for table in response["RouteTables"]]
+
+
+def gather_system_report(cluster, request, scheduler_commands, remote_command_executor):
```

Comment:
Let's add a docstring for this public utility/function.

Reply:
Done